### PR TITLE
add README docs for history

### PR DESCRIPTION
**Pull request by talIguaz**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -79,6 +80,7 @@ The `Client` class features the following methods for supporting operations on a
 - [`write`](#method-write) &mdash; writes data from pandas DataFrames to a collection.
 - [`execute`](#method-execute) &mdash; executes a backend-specific command on a collection.
   Each backend may support multiple commands.
+- [`history`](#method-history) &mdash; returns usage logs for frames service.
```
Comment (dinal) [inline README.md:83]: maybe something like - 
"returns information on every command ran with frames service"

File: README.md [hunk 2]
```diff
@@ -850,6 +852,96 @@ The following `execute` commands are specific to the `stream` backend; for more
                        "client_info": "my custom info", "partition": "PK1"})
   ```
 
+<a id="method-history"></a>
+### `history` Method
+
+By default every command ran with frames is logged.  
+`History` returns usage history logs for frames service into a pandas DataFrame.
```
Comment (dinal) [inline README.md:859]: same here

File: README.md [hunk 3]
```diff
@@ -850,6 +852,98 @@ The following `execute` commands are specific to the `stream` backend; for more
...
+<a id="method-history-params"></a>
+#### Common `history` Parameters
+
+- <a id="method-history-param-backend"></a>**backend** &mdash; filter logs by specific backend.
+
+  - **Type:** `str`
+  - **Requirement:** Optional
+
+- <a id="method-history-param-container"></a>**container** &mdash; filter logs by specific container.
+  - **Type:** `str`
```
Comment (gtopper) [inline README.md:883]: Do be consistent with vertical spacing.

File: README.md [hunk 4]
```diff
@@ -850,6 +852,98 @@ The following `execute` commands are specific to the `stream` backend; for more
...
+  - **Requirement:** Optional
+
+- <a id="method-history-param-table"></a>**table** &mdash; filter logs by specific table.
```
Comment (gtopper) [inline README.md:886]: by specific table -> by table

File: README.md [hunk 5]
```diff
@@ -850,6 +852,98 @@ The following `execute` commands are specific to the `stream` backend; for more
...
+  - **Type:** `str`
+  - **Requirement:** Optional
+
+- <a id="method-history-param-user"></a>**user** &mdash; filter logs by the user that executed the command.
+
+  - **Type:** `str`
+  - **Requirement:** Optional
+
+- <a id="method-history-param-action"></a>**action** &mdash; filter logs by specific frames action.
```
Comment (gtopper) [inline README.md:895]: specific frames action -> frames action

**Pull request merged**